entrypoint.sh

```diff
@@ -27,13 +27,23 @@ list_releases() {
     curl -s "https://api.github.com/repos/${REPO}/releases"
   fi
 }
+
+# Return version list from tags(refs/tags/vA.B.C and vA.B.C^{}) or releases
+list_versions() {
+  if [ "${INPUT_TAG:-false}" = "true" ]; then
+    git ls-remote -q --tags "https://github.com/${REPO}.git" | awk '{ print $2 }'
+  else
+    list_releases |  jq -r '.[] | .tag_name'
+  fi
+}
+
 LATEST_VERSION="$(\
-  list_releases | \
-  jq -r '.[] | .tag_name' | \
-  grep -oP '\d+\.\d+(\.\d+)?(-[^'\''\"\s]*)?$' | \
+  list_versions | \
+  grep -oP '\d+\.\d+(\.\d+)*(-[^'\''\"\s]*)?$' | \
```

Comment:
oh, wait. I noticed that you changed the regex too. 
Can you revert the change or explain why you updated the regex?

Reply:
## Why

Some tags have been given a small version that deviates from the rules of semver, so I changed it.

## Validity of change

Because, previouse commit split release(semver valid) and tag (semver deviate).

First, I think non-semver ruling tag(x.y.z.?) ignoring rule...but test case support non-semver(for -hoge suffix).
And I tried to support it.

Reply:
Hmm... I guess I don't get it. Can you list actual real-world examples who uses `x.y.z.?`?

Also, this repo doesn't intend to support non-semver in the first place, so can you propose a change separately?

Reply:
Note that `-xyz suffix` is actually valid semver. https://semver.org/#spec-item-10

Reply:
Right, example

https://github.com/thinca/vim-themis/releases/tag/v1.5.2.1

and get

```
5.2.1
```

Reply:
Oh, I get it. Hmm... 1.5.2.1 is a super weird tag name...

Ideally, I'd reject such inputs, but maybe it's fine to accept. I left a suggestion to make the regex simpler but LGTM other than that.

Reply:
Agree.
x.y.z.a ... can be simply drop away.